native/native.tests/testData/framework/objcexport/nativeEnum.kt

```diff
@@ -0,0 +1,10 @@
+package nativeEnum
+
+import kotlin.native.ObjCEnum
+import kotlin.experimental.ExperimentalObjCEnum
+
+@OptIn(kotlin.experimental.ExperimentalObjCEnum::class)
+@ObjCEnum("OBJCFoo")
```

Comment:
It can also specify `swiftName`, to test that the generated attribute is properly propagated to Swift.